README.md

```diff
@@ -28,259 +28,35 @@ different networks, you can read more about this in [this article](https://tezos
 
 ## Table of contents
 
-* [Static linux binaries](#static-linux)
-* [Native Ubuntu packages](#ubuntu)
-  * [Ubuntu packages on Raspberry Pi OS](#raspberry)
-* [Native Fedora packages](#fedora)
-* [Other linux](#linux)
-* [Brew tap for macOS](#macos)
-* [Systemd services for Tezos binaries](#systemd)
-* [Building instructions](#building)
-* [Setting up baking instance on Ubuntu](#baking-on-ubuntu)
+* [Getting binaries](#getting-binaries).
+* [Systemd services with Tezos binaries](#systemd).
+* [Launchd services with Tezos binaries](#systemd).
```

Comment:
I'm thinking that perhaps the `Launchd` document should be moved inside the `macOS` one.

From its content we only provide the former in the formulae for the latter, plus it makes sense to me that only `macOS` users might be interested in it.

For `sysyemd` packages given that they are provided for both native releases and can be build from scratch I'd leave those instructions in a separate doc (as they are now), but maybe instead of linking it from here we could add a link in the two distro docs as well as under the "other distros" section below?